inductiva/molecules/protein_solvation/README.md

```diff
@@ -25,13 +25,13 @@ The user can further define the following simulation parameters:
                 position Langevin dynamics.
     Default value if "md".
 - `n_steps_min`: Number of steps for energy minimization. Default value is 5000.
+- `ignore_warnings`: Whether to ignore warnings during grompp (gromacs preprocessor) step. If set to False, the simulation will fail if there are warnings. If True, the simulation will run and ignore the warnings, but the results may have inaccuracies. Use with caution. 
```

Comment:
If this is just during pre-processing, should the name of the argument state that?

Reply:
You call grompp several times in our commands, each time you make a energy minimization. So I wouldn't call it ignore_warnings_pre_processing, maybe ignore_grompp_warnings?

Reply:
I understand, but as an argument name it is rather unclear... ignore_grompp is to specific since that is not passed to the user either. Do what you feel it is best.